benchmark/benchmark_manycomps.py

```diff
@@ -4,23 +4,24 @@
 from openmdao.api import Problem, Group
 from openmdao.test_suite.build4test import DynComp, create_dyncomps
 
+
 class BM(unittest.TestCase):
     """Setup of models with lots of components"""
 
     def benchmark_100(self):
-        p = Problem(root=Group())
-        create_dyncomps(p.root, 100, 10, 10, 5)
+        p = Problem()
```

Comment:
Wonder if we need a way to "run all" on the benchmark suite before we pull-request.

Reply:
`testflo -bv`

Reply:
oh yeah, the old benchflow